MemeMe/MemeDetailViewController.swift

```diff
@@ -78,101 +74,84 @@ class ViewController: UIViewController, UIImagePickerControllerDelegate, UINavig
         dismiss(animated: true, completion: nil)
     }
 
-    // MARK: Image Picker Controller Did Cancel.
     // For when you hit the cancel button.
     func imagePickerControllerDidCancel(_: UIImagePickerController) {
         dismiss(animated: true, completion: nil)
     }
 
-    // MARK: - Keyboard Will Show.
     @objc func keyboardWillShow(_ notification: Notification) {
         if bottomTextField.isEditing {
             view.frame.origin.y = -getKeyboardHeight(notification)
         }
     }
 
-    // MARK: Get keyboard height.
     func getKeyboardHeight(_ notification: Notification) -> CGFloat {
         let userInfo = notification.userInfo
         let keyboardSize = userInfo![UIResponder.keyboardFrameEndUserInfoKey] as! NSValue // of CGRect
         return keyboardSize.cgRectValue.height
     }
 
-    // MARK: Keyboard Will Hide.
     @objc func keyboardWillHide(_ notification: Notification) {
         view.frame.origin.y = 0
     }
 
-    // MARK: - 0
     func textFieldShouldClear(_ textField: UITextField) -> Bool {
         return true
     }
 
-    // MARK: TextField Did Begin Editing.
     func textFieldDidBeginEditing(_ textField: UITextField) {
         // if textField's text equals Top or textField's text equals Bottom then set textField's text to ""
         if textField.text == "TOP" || textField.text == "BOTTOM" {
             textField.text = ""
         }
     }
 
-    // MARK: TextField Should Return.
     func textFieldShouldReturn(_ textField: UITextField) -> Bool {
         textField.resignFirstResponder()
         view.endEditing(true)
         return true
     }
 
-    // MARK: - Meme Text Attributes.
     let memeTextAttributes: [NSAttributedString.Key: Any] = [
         NSAttributedString.Key.strokeColor: UIColor.black,
         NSAttributedString.Key.foregroundColor: UIColor.white,
         NSAttributedString.Key.font: UIFont(name: "HelveticaNeue-CondensedBlack", size: 40)!,
         NSAttributedString.Key.strokeWidth: -3.0
     ]
 
-    // MARK: Subscribe to keyboard notifications.
+    // MARK: Subscribe & Unsubscribe to keyboard notifications
     func subscribeToKeyboardNotifications() {
         NotificationCenter.default.addObserver(self, selector: #selector(keyboardWillShow(_:)), name: UIResponder.keyboardWillShowNotification, object: nil)
         NotificationCenter.default.addObserver(self, selector: #selector(keyboardWillHide(_:)), name: UIResponder.keyboardWillHideNotification, object: nil)
     }
 
-    // MARK: Unsubscribe to keyboard notifications.
     func unsubscribeFromKeyboardNotifications() {
         NotificationCenter.default.removeObserver (self, name: UIResponder.keyboardWillShowNotification, object: nil)
         NotificationCenter.default.removeObserver(self, name: UIResponder.keyboardWillHideNotification, object: nil)
     }
 
-    // MARK: - Save the meme.
+    // MARK: Create and Save the meme.
     func save() {
-        // Create the meme.
+        // Update the meme.
```

Comment:
This says update the meme but it appears to just save the new meme, does not appear to "Update" anything.